onadata/apps/api/management/commands/export_inactive_users_orgs.py

```diff
@@ -0,0 +1,860 @@
+                    key_parts = obj["Key"].split("/")
+                    if len(key_parts) > 1:
+                        # Skip username, get first folder
+                        folder = key_parts[1]
+                        folder_sizes[folder] = folder_sizes.get(folder, 0) + size
+
+        # Convert to MB and format
+        total_size_mb = round(total_size / (1024 * 1024), 2)
+        folder_sizes_mb = {
+            folder: round(size / (1024 * 1024), 2)
+            for folder, size in folder_sizes.items()
+        }
+
+        return {
+            "total_size_mb": total_size_mb,
+            "storage_breakdown": json.dumps(folder_sizes_mb),
+        }
+
+    except (BotoCoreError, ClientError) as e:
+        return {"total_size_mb": 0, "storage_breakdown": "{}", "error": str(e)}
+
+
+def _process_user_directory(user_dir):
+    """Helper function to process directory and calculate sizes"""
+    folder_sizes = {}
+    total_size = 0
+
+    # Walk through all files in user directory
+    for root, _dirs, files in os.walk(user_dir):
+        for file in files:
+            file_path = os.path.join(root, file)
+            try:
+                file_size = os.path.getsize(file_path)
+                total_size += file_size
+
+                # Determine folder category based on path
+                relative_path = os.path.relpath(root, user_dir)
+                if relative_path == ".":
+                    folder_name = "root"
+                else:
+                    folder_name = relative_path.split(os.sep)[0]
+
+                if folder_name not in folder_sizes:
+                    folder_sizes[folder_name] = 0
+                folder_sizes[folder_name] += file_size
+            except OSError:
+                # Skip files we can't read
+                continue
+
+    return total_size, folder_sizes
+
+
+def calculate_filesystem_storage_size(storage_container, username):
+    """Calculate storage size for filesystem backend as standalone function"""
+    if not storage_container:
+        return {"total_size_mb": 0, "storage_breakdown": "{}"}
+
+    try:
+        user_dir = os.path.join(storage_container, username)
+        if not os.path.exists(user_dir):
+            return {"total_size_mb": 0, "storage_breakdown": "{}"}
+
+        total_size, folder_sizes = _process_user_directory(user_dir)
+
+        # Convert to MB
+        total_size_mb = total_size / (1024 * 1024)
+        folder_sizes_mb = {
+            folder: size / (1024 * 1024) for folder, size in folder_sizes.items()
+        }
+
+        return {
+            "total_size_mb": round(total_size_mb, 2),
+            "storage_breakdown": json.dumps(folder_sizes_mb),
+        }
+
+    except (OSError, IOError) as e:
+        return {"total_size_mb": 0, "storage_breakdown": "{}", "error": str(e)}
+
+
+# pylint: disable=too-many-instance-attributes
+class Command(BaseCommand):
+    """Export inactive users and organizations (2+ years of inactivity)"""
```

Comment:
Could we add usage documentation as part of the docstrings? Most commands we've added usage section/how to run the command